src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -153,11 +159,15 @@ public void testInit() {
 
     @Override
     public void robotPeriodic() {
+        Logger.updateEntries();
         CommandScheduler.getInstance().run();
     }
 
     @Override
     public void autonomousInit() {
+
+        // subsystems.drivebaseSubsystem.resetPose(RigidTransform2.ZERO);
+
```

Comment:
Delete

Reply:
We accidentally commented that out. We think it's Jonah's, so we uncommented out and left it in.